lib/stripe/api_operations/request.rb

```diff
@@ -12,7 +12,7 @@ def request(method, url, params={}, opts={})
           api_base = headers.delete(:api_base)
           # Assume all remaining opts must be headers
 
-          response, opts[:api_key] = Stripe.request(method, url, api_key, params, headers, api_base)
+          resp, opts[:api_key] = Stripe.request(method, url, api_key, params, headers, api_base)
```

Comment:
These are nitpicky changes, but I tried to call all local variables containing a `StripeResponse` `resp` for global consistency.